otter/test/convergence/test_effecting.py

```diff
@@ -112,3 +114,36 @@ def test_multiple_requests_of_different_type(self):
                              headers=None,
                              data=data_sentinel)]
         self.assertCompileTo(conv_requests, expected_effects)
+
+
+@implementer(IStep)
+class Steppy(object):
+    """A dummy step."""
+    def as_request(self):
+        """Return a simple GET .../whatever on CLOUD_LOAD_BALANCERS"""
+        return Request(service=ServiceType.CLOUD_LOAD_BALANCERS,
+                       method="GET",
+                       path="whatever")
+
+
+class StepsToEffectTests(SynchronousTestCase):
```

Comment:
This seems to be testing `_reqs_to_effect` indirectly. If so, then are `RequestToEffectTests` required?